### PR TITLE
Disable colour picker UI by default - Issue #57

### PR DESCRIPTION
**Pull request by AllanOcelot**
This addresses long standing item #57 - and is hopefully the start of me adding more user options in future 😁  .

- Added in option to display the colour picker, by default the colour picker will be hidden
- Added in translation string for new option

### CODE REVIEW

File: src/libs/state.js [hunk 1]
```diff
@@ -69,6 +69,7 @@ const stateObj = {
             state_key: 'kiwi-state',
         },
         showAutocomplete: true,
+        showColour: false,
```
Comment (prawnsalad) [inline src/libs/state.js]: Shouldn't this be `showColourPicker`?

File: src/components/ControlInput.vue [hunk 2]
```diff
@@ -133,6 +134,10 @@ export default {
                 this.active_tool = tool;
             }
         },
+        //Check the users settings to see if they have turned on the colour picker
+        displayColourPicker: function displayColourPicker() {
+           this.display_colour_picker = state.setting('showColourPicker');
+        },
```
Comment (prawnsalad) [inline src/components/ControlInput.vue]: This `displayColourPicker()` function would be better suited in the computed section of the component and the `display_colour_picker` data property removed. eg:
~~~javascript
shouldDisplayColourPicker() {
    return state.setting('showColourPicker');
}
~~~

You would then use it like `v-if="shouldDisplayColourPicker"`

**Comment by prawnsalad** [comment]
The app.en-US.po translation file should not be modified. The translations.kiwiirc.com website handles all that for us based on the app.dev.po file.

**Comment by AllanOcelot** [comment]
Outdated with the new version of Kiwi.

**Pull request closed without merging**